templates/repo/actions/workflow_dispatch_inputs.tmpl

```diff
@@ -0,0 +1,23 @@
+{{range $item := .WorkflowDispatchConfig.Inputs}}
+<div class="ui field {{if .Required}}required{{end}}">
+	{{if eq .Type "choice"}}
+		<label>{{.Description}}:</label>
+		<select class="ui selection type dropdown" name="{{.Name}}">
+			{{range .Options}}
+			<option value="{{.}}" {{if eq $item.Default .}}selected{{end}} >{{.}}</option>
+			{{end}}
+		</select>
+	{{else if eq .Type "boolean"}}
+		<div class="ui inline checkbox">
+			<label>{{.Description}}</label>
```

Comment:
Sometimes there is `:` sometimes there is not.

Maybe we should remove all `:`from that label.